chat/fe/src/apis/RoomApi.ts

```diff
@@ -47,6 +50,10 @@ export async function initializeRoomApi() {
       }
     }
   });
+
+  socket.on('delete-message', ({ room, id }) => {
+    deletingMessageListener[room](id);
```

Comment:
If I'm reading correctly, this would probably throw an error if someone deletes a message in a room you don't have access to.  We should check for existence of `deletingMessageListener[room]`. https://github.com/adamjberg/engram/issues/176

Reply:
Someone? not sure I'm following this one